who-we-are/index.md

```diff
@@ -11,11 +11,19 @@ redirect_from:
 
 # Meet us
 
-The Foundation for Public Code was founded by Ben Cerveny, Boris van Hoytema and Arnout Schuijff in 2019 to guide international collaboration among public organizations around codebase design, quality and governance.
+The Foundation for Public Code is a non-profit association whose mission is 'to enable public-purpose software and policy that is open and collaborative.'
 
-Leveraging 30 combined years of experience working with cities and technology-enabled collaboration, Boris and Ben recognized policy and whitepapers were ineffective catalysts in this space. Their landscape analysis revealed that stewardship was the key to converting the massive public investment in open source and collaboration into long-term, sustainable implementation.
+We do this by:
 
-Based on their experience of technological innovation in a traditional industry, Pieter van der Does and Arnout Schuijff (co-founders of the Adyen payments platform) agreed with Ben and Boris that public organizations deserved better software. They joined the Board of Directors to share their expertise and connections.
+* stewarding public purpose, open source codebases
+* sharing our industry-leading expertise
+* offering tailored training for public organizations
+* developing tools to make it easier for public organizations to build and maintain software together
+* facilitating collaborative software development and maintenance for public organizations
+
+Ben Cerveny, Boris van Hoytema and Arnout Schuijff founded the Foundation in 2019 in Amsterdam, based on their realization that stewardship was the key to converting the massive public investment in open source and collaboration into long-term, sustainable implementation.
```

Comment:
```suggestion
Ben Cerveny, Boris van Hoytema and Arnout Schuijff founded the Foundation in 2019, based on their realization that stewardship was the key to converting the massive public investment in open source and collaboration into long-term, sustainable implementation. We were founded  in Amsterdam, with and aim to operate globally. 
```
Just a suggestion to clarify both whose realization it was (not gemeente Amsterdam) and the ambition.